TTHAnalysis/macros/submit.py

```diff
@@ -0,0 +1,230 @@
+import os
```

Comment:
@sscruz the idea was to have it in the susyMaintenance stuff, but for me it can be left here without any problem.

Perhaps let's ping @nachosandres , since he wrote the bulk of the script.